lib/api/fields/types/views/field-description.php

```diff
@@ -11,4 +11,4 @@
 ?>
 
 <br /><a class="bs-read-more" href="#"><?php esc_html_e( 'More...', 'tm-beans' ); ?></a>
-<div class="bs-extended-content" style="display: none;"><?php echo $extended; // @codingStandardsIgnoreLine - WordPress.XSS.EscapeOutput.OutputNotEscaped - To optimize, escaping is handled in the calling function. ?></div>
+<div class="bs-extended-content" style="display: none;"><?php echo $extended; ?></div><?php // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped -- To optimize, escaping is handled in the calling function. ?>
```

Comment:
I'd rather see the escaping done as late as possible. Someone shouldn't need to go and find all the places where this file might be called from, to check if a variable is escaped or not.

Reply:
I agree.  But in this case, doing so would have required us to call `wp_kses_post` twice instead of once.  For performance, we opted to call it once in the calling function.

Explanation:

The description can have `<!--more-->` in it.  When it does, we split that description into 2 parts: description and extended.  The original description is run through `wp_kses_post()` before it's split. We did that for performance.  

BUT I'll be running some benchmarks when I run this API through the micro-profiler.  I'm hoping that the impact is small enough that we can escape late inside of the view file. YTD.